modules/commsInterface/commsInterface.py

```diff
@@ -0,0 +1,74 @@
+import usb.core
+import usb.util
+import serial
```

Comment:
Please add a comment stating this is Pyserial, a requirements.txt file would be good as well so someone can just pip install the dependencies inside the text file